pkg/api/api.go

```diff
@@ -1,6 +1,7 @@
 package api
 
 import (
+	"encoding/json"
```

Comment:
Missing go format in the imports, actually just add an empty line to separate standard/non-standard pkgs
```go
import (
  "encoding/json"

  "github.com/asaskevich/govalidator"
  "github.com/hellofresh/janus/pkg/proxy"
)
```